content/install/server/ssl.es.md

```diff
@@ -0,0 +1,88 @@
++++
+date = "2017-04-15T14:39:04+02:00"
+title = "Configurar SSL"
+url = "configure-ssl"
+
+[menu.install]
+  identifier = "configure-ssl"
+  parent = "install_server"
+  weight = 10
++++
+
+Drone soporta configuración ssl montando certificados en tu contenedor. Nota que también existe soporte para [Lets Encrypt]({{< relref "lets_encrypt.md" >}}) automatizado.
+
+```diff
+services:
+  drone-server:
+    image: drone/drone:{{% version %}}
+    ports:
++     - 80:80
++     - 443:443
+      - 9000:9000
+    volumes:
+      - /var/lib/drone:/var/lib/drone/
++     - /etc/certs/drone.foo.com/server.crt:/etc/certs/drone.foo.com/server.crt
++     - /etc/certs/drone.foo.com/server.key:/etc/certs/drone.foo.com/server.key
+    restart: always
+    environment:
++     - DRONE_SERVER_CERT=/etc/certs/drone.foo.com/server.crt
++     - DRONE_SERVER_KEY=/etc/certs/drone.foo.com/server.key
+```
+
+Actualiza tu configuración para exponer los siguientes puertos:
+
+```diff
+services:
+  drone-server:
+    image: drone/drone:{{% version %}}
+    ports:
++     - 80:80
++     - 443:443
+      - 9000:9000
+```
+
+Actualiza tu configuración para montar tu certificado y llave:
+
+```diff
+services:
+  drone-server:
+    image: drone/drone:{{% version %}}
+    ports:
+      - 80:80
+      - 443:443
+      - 9000:9000
+    volumes:
+      - /var/lib/drone:/var/lib/drone/
++     - /etc/certs/drone.foo.com/server.crt:/etc/certs/drone.foo.com/server.crt
++     - /etc/certs/drone.foo.com/server.key:/etc/certs/drone.foo.com/server.key
+```
+
+Actualiza tu configuración para proveer las rutas de acceso de tu certificado y llave:
+
+```diff
+services:
+  drone-server:
+    image: drone/drone:{{% version %}}
+    ports:
+      - 80:80
+      - 443:443
+      - 9000:9000
+    volumes:
+      - /var/lib/drone:/var/lib/drone/
+      - /etc/certs/drone.foo.com/server.crt:/etc/certs/drone.foo.com/server.crt
+      - /etc/certs/drone.foo.com/server.key:/etc/certs/drone.foo.com/server.key
+    restart: always
+    environment:
++     - DRONE_SERVER_CERT=/etc/certs/drone.foo.com/server.crt
++     - DRONE_SERVER_KEY=/etc/certs/drone.foo.com/server.key
+```
+
+# Cadena de certificados
+
+El problema más común es proveer un certificado sin la cadena intermedia.
```

Comment:
IMHO I would change "intermedia" for "intermediaria" since it's the chain from the in-between company providing the cert.